src/cdi/CPEloss.hh

```diff
@@ -126,10 +116,37 @@ public:
   virtual size_t getNumEnergies() const = 0;
 
   /*!
-	 * \brief Returns the general Eloss model type (Analytic),
-	 * defined in the enum at the top of this file.
-	 */
-  virtual rtt_cdi::CPModelType getModelType() const = 0;
+   * \brief Query to determine the transporting particle type.
+   */
+  CParticle getProjectile() const { return projectile; }
+
+  /*!
+   * \brief Query to determine the target species type.
+   */
+  CParticle getTarget() const { return target; }
+
+  /*!
+   * \brief Query to determine the CP Model type.
+   */
+  CPModelType getModelType() const { return model_type; }
+
+  /*!
+   * \brief Query to determine the angle cutoff used by the CP model.
+   */
+  CPModelAngleCutoff getModelAngleCutoff() const { return model_angle_cutoff; }
```

Comment:
Please add a unit test for this function.

Reply:
Done.